.github/workflows/build.yml

```diff
@@ -1,7 +1,3 @@
-on:
-  schedule:
-    - cron: "0 00,12 * * *"
```

Comment:
This is to stop build minutes issue? I saw some convo in discord.

I wonder if rather than delete, we change it to run once every 24 hours? or less depending on limit calculation - like mitigate, but don't stop altogether?

Reply:
You will have to take that up with @kimadactyl